### DISCUSSION TITLE
How to handle queries with multiple return values

### DISCUSSION THREAD

**Discussion by mephenor** [SBRG/ModelPolisher#111] [Release 2.1]
For some ids getFomula and getCharge return a ResultSet with more than one entry (possibly for other queries too).
Some investigation is needed as to whether this entries are unique.
However, if these are indeed different results, how should we deal with them?

**Comment by mephenor** [comment]
As a general case, only unique values returned from queries are processed now and in case of ambiguity, the results are dropped.
For gene reaction rules, as discussed in another issue (#90 ) the resulting associations are merged under an OR.

**Comment by Schmoho** [comment]
Moving this to a discussion as it does not really state a resolvable issue or feature request.

To me, this primarily raises the point that the ModelPolisher is lacking a comprehensive documentation of what it actually does to a model.
